lib/yoti/data_type/base_profile.rb

```diff
@@ -8,7 +8,9 @@ class BaseProfile
     #
     # @return [Hash{String => Yoti::Attribute}]
     #
-    attr_reader :attributes
+    def attributes
```

Comment:
Do we also need `def attributes_list` (or similar) that would return the full list of `Array<Yoti::Attribute>` like other SDKs?